Monofoxe/Monofoxe.Engine/ColorExtensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+
+
+namespace Monofoxe.Engine
+{
+	/// <summary>
+	/// <see cref="Color"/> extensions
+	/// </summary>
+	public static class ColorExtensions
+	{
+
+		/// <summary>
+		/// Return the HsvColor equivalent of a given color 
+		/// </summary>
+		/// <param name="color">The color that will be converted</param>
```

Comment:
Remove param and returns, leave only the summary. Though, I'd remove it also, no need for comments just for the sake of commenting.